<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yeniUrun</class>
 <widget class="QDialog" name="yeniUrun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget {
	background-color: white;
}

QLabel {
	
	color: rgb(0, 0, 0);
}


QPushButton {
	background-color: black;
    color: white; 
    border: 2px solid rgb(0, 0, 0);
    border-radius: 10px; 
    padding: 10px 20px; 
    font-size: 13px; 
    font-weight: bold;
}

QPushButton:hover {
	background-color: white; 
    border-color: black;
	color: black;
}

QPushButton:pressed {
    background-color:   black; 
    border-color: white;
	color: white; 
}

QLineEdit {
    background-color: #ecf0f1; 
    color: #2c3e50; 
    border: 2px solid #bdc3c7; 
    border-radius: 10px; 
    padding: 8px 12px; 
    font-size: 12px; 
}

QLineEdit:focus {
    border-color: #2c3e50; 
    background-color: #ffffff; 
}

QLineEdit:hover {
    border-color: #34495e; 
}

/*-----------------------*/

QComboBox {
    background-color: black; 
   	border: 2px solid black;
	padding: 10px 20px; 
    border-radius: 8px; 
    padding: 4px; 
    color: white; 
    font-size: 16px; /* حجم الخط */
    combobox-popup: 0; /* منع ظهور قائمة منبثقة خارجية */
}

QComboBox:hover {

   	border: 2px solid white;

}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 25px; /* عرض زر القائمة المنسدلة */
    border-left: 1px solid white; /* خط فاصل بين القائمة المنسدلة وبقية الصندوق */
}

QComboBox::down-arrow {
    image: url(:/other ic/icon/other ico/arrow down.png);
    width: 13px;
    height: 13px;
}

QComboBox::down-arrow:on {
	image: url(:/other ic/icon/other ico/arrow up.png)
}

QComboBox QAbstractItemView {
    background-color: black; /* لون خلفية القائمة المنسدلة */
   
    selection-background-color: black; /* لون خلفية العنصر المحدد */
    selection-color: white; /* لون النص للعناصر المحددة */
    color: white; /* لون النص للعناصر غير المحددة */
}

</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_15">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Yeni bir ürün ekle</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QGridLayout" name="gridLayout_11">
          <property name="horizontalSpacing">
           <number>20</number>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_9">
            <property name="verticalSpacing">
             <number>13</number>
            </property>
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <fontweight>DemiBold</fontweight>
                 </font>
                </property>
                <property name="text">
                 <string>Ürün adı</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLineEdit" name="urunAd">
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <fontweight>DemiBold</fontweight>
                 </font>
                </property>
                <property name="text">
                 <string>Ürün fiyatı</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLineEdit" name="urunFiyat">
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <widget class="QLabel" name="label_8">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <fontweight>DemiBold</fontweight>
                 </font>
                </property>
                <property name="text">
                 <string>Urunun Maliyeti</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLineEdit" name="urunMaliyeti">
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_10">
            <property name="verticalSpacing">
             <number>13</number>
            </property>
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QLabel" name="label_6">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <fontweight>DemiBold</fontweight>
                 </font>
                </property>
                <property name="text">
                 <string>Ürün barkodu</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLineEdit" name="urunBarkod">
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="0" column="0">
               <widget class="QLabel" name="label_9">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <fontweight>DemiBold</fontweight>
                 </font>
                </property>
                <property name="text">
                 <string>Ürün kategorisi</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QComboBox" name="comboBox_Kategori">
                <item>
                 <property name="text">
                  <string>Kategori seçiniz</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item row="2" column="0">
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <widget class="QLabel" name="label_5">
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                  <fontweight>DemiBold</fontweight>
                 </font>
                </property>
                <property name="text">
                 <string>Stokta miktarı</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLineEdit" name="urunMiktar">
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_durumu">
          <property name="font">
           <font>
            <pointsize>11</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::LeftToRight</enum>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white;</string>
          </property>
          <property name="text">
           <string>Lütfen tüm alanları doldurun</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <layout class="QGridLayout" name="gridLayout_14">
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_12">
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton_2">
              <property name="minimumSize">
               <size>
                <width>85</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
	background-color: red;
    color: white; 
    border: 2px solid red;
    border-radius: 10px; 
    padding: 8px 13px; 
    font-size: 16px; 
    font-weight: bold;
}

QPushButton:hover {
	background-color: white; 
    border-color: red;
	color: red;
}

QPushButton:pressed {
    background-color:   red; 
    border-color: white;
	color: white; 
}</string>
              </property>
              <property name="text">
               <string>iptal</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton">
              <property name="minimumSize">
               <size>
                <width>85</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
	background-color: rgb(147, 170, 0);
    color: white; 
    border: 2px solid rgb(147, 170, 0);
    border-radius: 10px; 
    padding: 8px 13px; 
    font-size: 16px; 
    font-weight: bold;
}

QPushButton:hover {
	background-color: white; 
    border-color: rgb(147, 170, 0);
	color: rgb(147, 170, 0);
}

QPushButton:pressed {
    background-color:   rgb(147, 170, 0); 
    border-color: white;
	color: white; 
}</string>
              </property>
              <property name="text">
               <string>Ekle</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
